services/ows_refactored/prod/landsat/ows_s2_cfg.py

```diff
@@ -0,0 +1,46 @@
+from prod.landsat.style_ls_cfg import styles_s2_list
```

Comment:
Why is Sentinel-2 sitting under landsat in the hierarchy?

Reply:
they share the same styles, i dont know why

Reply:
renamed to `surface_reflectance`